modules/ROOT/pages/_partials/include-alerts-config.adoc

```diff
@@ -29,7 +29,7 @@ You can configure the condition under which an alert is triggered.
 |===
 2+| Settings for Alerts
 | Name | A title for your alert.
-| Condition | WHEN: For triggering an alert when the `JVM - CPU % Utilization` is above a specified level.
+| `Condition` | WHEN: For triggering an alert when the `JVM - CPU % Utilization` is above a specified level.
 |===
```

Comment:
These will be translated if we don't mark them as bold or code. Stacey agreed we'd go with code for "field values"...